dgraph/cmd/alpha/metrics_test.go

```diff
@@ -176,7 +176,7 @@ func TestMetrics(t *testing.T) {
 
 	requiredMetrics := []string{
 		// Go Runtime Metrics
-		"go_goroutines", "go_memstats_gc_cpu_fraction", "go_memstats_heap_alloc_bytes",
```

Comment:
Why are we removing this?

Reply:
The new version of the library removes this as explained in the description. Because we are upgrading the library, we have to remove this metric.